test/integration/features/basic.feature

```diff
@@ -147,6 +147,12 @@ Feature: Basic
      """
      exposed
      """
+
+  Scenario: User can access ruby-ex routing url
+   Given Minishift has state "Running"
+     When status code of HTTP request to "Routing" is equal to "200"
+     Then body of HTTP request to "Routing" contains "ok"
```

Comment:
These steps are executed in namespace ```ruby``` and these steps are derived from same function ```verifyHTTPResponse```. Will check how it will be used in more specific way.